include/wx/generic/private/listctrl.h

```diff
@@ -313,7 +313,18 @@ class wxListLineData
                            int width);
 };
 
-WX_DECLARE_OBJARRAY(wxListLineData, wxListLineDataArray);
+class wxListLineDataArray : public wxVector<wxListLineData*>
+{
+public:
+    void Clear()
+    {
+        for ( size_t n = 0; n < size(); ++n )
+            delete(operator[](n));
```

Comment:
This looks a bit weird, I'd write it either as `at(n)` (which is a bit less efficient due to the index check) or `(*this)[n]`.

Reply:
Fixed. This came from wxBaseObjectArray::DoEmpty()